http/src/main/resources/init-resources/msdsvmcontent/ms_dsvm_script_v5_cat.sh

```diff
@@ -33,8 +35,29 @@ SERVER_APP_IP=''
 
 # Start Jupyter server with custom parameters
 
-/anaconda/bin/jupyter server --ServerApp.port=$SERVER_APP_PORT --ServerApp.token=$SERVER_APP_TOKEN --autoreload --ServerApp.ip=$SERVER_APP_IP --allow-root >/dev/null 2>&1&
+/anaconda/bin/jupyter server \
+--ServerApp.certfile=$SERVER_APP_CERTFILE \
+--ServerApp.keyfile=$SERVER_APP_KEYFILE \
+--ServerApp.port=$SERVER_APP_PORT \
+--ServerApp.token=$SERVER_APP_TOKEN \
+--ServerApp.ip=$SERVER_APP_IP \
+--ServerApp.base_url=$SERVER_APP_BASE_URL \
+--ServerApp.websocket_url=$SERVER_APP_WEBSOCKET_URL \
+--ServerApp.allow_origin=$SERVER_APP_ALLOW_ORIGIN \
+--autoreload --allow-root >/dev/null 2>&1&
 
 # Store Jupyter Server parameters for reboot process
 
-sudo crontab -l 2>/dev/null| cat - <(echo "@reboot /anaconda/bin/jupyter server --ServerApp.port=$SERVER_APP_PORT --ServerApp.token=$SERVER_APP_TOKEN --autoreload --ServerApp.ip=$SERVER_APP_IP --allow-root >/dev/null 2>&1&") | crontab -
+sudo crontab -l 2>/dev/null| cat - <(echo "@reboot /anaconda/bin/jupyter server --ServerApp.certfile=$SERVER_APP_CERTFILE --ServerApp.keyfile=$SERVER_APP_KEYFILE --ServerApp.port=$SERVER_APP_PORT --ServerApp.token=$SERVER_APP_TOKEN --ServerApp.ip=$SERVER_APP_IP --ServerApp.base_url=$SERVER_APP_BASE_URL --ServerApp.websocket_url=$SERVER_APP_WEBSOCKET_URL --ServerApp.allow_origin=$SERVER_APP_ALLOW_ORIGIN --autoreload --allow-root >/dev/null 2>&1&") | crontab -
+
+# Login to ACR repo to pull the image for Relay Listener
+
+docker login terradevacrpublic.azurecr.io -u $6 -p $7
+
+#Run docker container with Relay Listener
+
+docker run -d --restart always --name RelayListener \
+--env LISTENER_RELAYCONNECTIONSTRING="Endpoint=sb://$1.servicebus.windows.net/;SharedAccessKeyName=$4;SharedAccessKey=$5;EntityPath=$2" \
+--env LISTENER_RELAYCONNECTIONNAME="$2" \
+--env LISTENER_TARGETPROPERTIES_TARGETHOST="http://$3:8080" \
+terradevacrpublic.azurecr.io/terra-azure-relay-listeners:53d7992
```

Comment:
the hash should come from input as well

Reply:
you mean docker version?

Reply:
no...the listener image version...`53d7992` part

Reply:
Yes. I meant image version.

Reply:
@Qi77Qi  I think we need to pass the full information not only image version.